dist/README.md

```diff
@@ -140,6 +140,20 @@ current configuration variables, if any, into the `~/.env` file. You
 may then inspect or modify your variables and re-run the bootstrap
 script again.
 
+## Additional configuration
+
+If you would like to pass any other environment variables to your
+Grist instance, you may use a supplemental file called `grist-env` to
+define any other extra variables. The syntax is the same as the
+standard `.env` file syntax.
+
+The configuration defined in `.env` takes precedence over the
+variables defined in `grist-env`.
+
+The extra variables defined in `grist-env` will only be applied to the
+Grist Docker Compose service. In particular, they will not affect
+Traefik's, Dex's, or Authelia's environment.
```

Comment:
Hmm the line between the two env files is a little tricky for things like `APP_HOME_URL`. Docker compose will want to expand that in a few places, so it needs to be in `.env` and can't go to `grist-env`. But it is a very common setting of Grist. I wonder if you need to specifically say to leave variables that are in .env where they are. Also moving say `TEAM` across would break things, right? Even though you might think of it as a setting of the Grist service.

Reply:
Yes, it's also complicated because the variable was called `TEAM` "externally" to make it easy but internally it's really called `GRIST_SINGLE_ORG`.

I think this is a very minor point of confusion, though. Out of the approximately one hundred variables documented in the README, only eight of these are defined by `.env`, and they're all user-controllable. If someone *really* has a problem with this state of affairs, I don't think directly editing `compose.yaml` should be too much trouble.

Is this reasonable, or do you think I need a better story for those eight variables?

Reply:
What if we were strict in the documentation that the variables in .env are exactly the ones that should be there, no more, no less? That's a rule a developer with no idea of how this soup works could follow. The language about precedence, while it may be accurate as far as it goes, kind of opens up a can of worms.

Try to imagine you hit your head and come in fresh to read these instructions. You try something, it doesn't quite work. What do you do next? There are two config files and you're not sure what the difference is between them. Maybe you put something in the wrong one? You move some across. You try again, and it still doesn't work.

Anything we can rule out or clarify or simplify will help.